crl/crl_x509/crl_test.go

```diff
@@ -441,6 +447,19 @@ func TestParseRevocationList(t *testing.T) {
 	if numCerts != expected {
 		t.Errorf("bad number of revoked certificates. got: %d want: %d", numCerts, expected)
 	}
+
+	// Check that 'thisUpdate' of 'GENERALIZEDTIME 20500706164338Z0000' (time
```

Comment:
Can you explicitly note that these two test cases are not included in upstream? I'm trying to make it super simple to keep track of where we differ from upstream. See example comment https://github.com/letsencrypt/boulder/blob/455cdade69e38ddd80f6563f692ee0d63ab83eb9/crl/crl_x509/crl.go#L249

Reply:
Oh, for sure!